Zeeguu Reader/Pods/Zeeguu-API-iOS/ZeeguuAPI/ZeeguuAPI/Bookmark.swift

```diff
@@ -49,6 +49,9 @@ public class Bookmark {
 	
 	// MARK: Properties -
 	
+	/// The id of this bookmark
+	public var id: String
```

Comment:
You think it should be int? Note: The feed id is also a String (from the beginning)...